scalecube-configuration-api/src/main/java/io/scalecube/configuration/api/ReadEntryHistoryRequest.java

```diff
@@ -1,7 +1,7 @@
 package io.scalecube.configuration.api;
 
 /** Represents a request o fetch data from a repository. */
-public class FetchRequest implements AccessRequest {
+public class ReadEntryHistoryRequest implements AccessRequest {
```

Comment:
Why `FetchRequest` renamed to `ReadEntryHistoryRequest`?
Shouldn't it be `ReadEntryRequest`?